geonode_mapstore_client/static/mapstore/configs/localConfig.json

```diff
@@ -2801,6 +2821,11 @@
                                     "labelId": "gnviewer.supplementalInformation",
                                     "value": "{context.get(state('gnResourceData'), 'supplemental_information')}"
                                 },
+                                {
```

Comment:
I don't think we need additional implementation because the date type supports already the range type with value as object with start and end parameter (see [DetailsInfo](https://github.com/GeoNode/geonode-mapstore-client/blob/master/geonode_mapstore_client/client/js/components/DetailsPanel/DetailsInfo.jsx#L101-L109))
Could you verify if the following configuration is working?
Please remove getTemporalExtent the If the following configuration is not working
```js
{
    "type": "date",
    "format": "YYYY-MM-DD HH:mm",
    "labelId": "gnviewer.temporalExtent",
    "disableIf": "{!context.get(state('gnResourceData'), 'temporal_extent_start') && !context.get(state('gnResourceData'), 'temporal_extent_end')}",
    "value": {
        "start": "{context.get(state('gnResourceData'), 'temporal_extent_start')}",
        "end": "{context.get(state('gnResourceData'), 'temporal_extent_end')}"
    }
}
```
It it's not working I would prefer we fix what missing in the date range configuration